### DISCUSSION TITLE
Form helper

### DISCUSSION THREAD

**Discussion by der-On** [bagisto/bagisto#5340] [Ideas]
Coming from other CMS/Shop frameworks I'm used to form helpers, that ease the creation of backend forms and standarsize form behaviour and make forms more update save.

Are there any plans on including some kind of form helper or form generation framework?
A very good example for creating forms easily can be seen in [WinterCMS](https://wintercms.com/docs/backend/forms) which is also based on Laravel.

**Comment by devansh-webkul** [comment]
Currently, there are no such plans, as we need to simplify the existing stuff first but in the future we will see this.
